packages/devtools/lib/src/main.dart

```diff
@@ -136,6 +140,44 @@ class PerfToolFramework extends Framework {
     screensReady.future.then(app.devToolsReady);
   }
 
+  void initSseConnection() {
+    // When running the debug DDC Build, the server won't be running so we
+    // can't connect to its API (for now at least, the API is optional).
+    if (isDebugBuild()) {
```

Comment:
we should provide a way to debug the server workflows in debug mode. File a bug for us to configure our environment to support that.

Reply:
I've opened https://github.com/flutter/devtools/issues/1036. It's not terribly bad as-is, my VS Code launch config automatically does a release build before running - so you can just press F5 and it'll do a release build, run the server, spawn the browser - ofc that's slower than if we were using the DDC one.